backend_new/test/integration/application.e2e-spec.ts

```diff
@@ -160,4 +175,796 @@ describe('Application Controller Tests', () => {
+        mobility: false,
+        vision: false,
+        hearing: false,
+      },
+      alternateContact: {
+        type: 'example type',
+        otherType: 'example other type',
+        firstName: 'example first name',
+        lastName: 'example last name',
+        agency: 'example agency',
+        phoneNumber: '111-111-1111',
+        emailAddress: 'example@email.com',
+        address: exampleAddress,
+      },
+      applicationsAlternateAddress: exampleAddress,
+      applicationsMailingAddress: exampleAddress,
+      listings: {
+        id: listing1Created.id,
+      },
+      demographics: {
+        ethnicity: 'example ethnicity',
+        gender: 'example gender',
+        sexualOrientation: 'example sexual orientation',
+        howDidYouHear: ['example how did you hear'],
+        race: ['example race'],
+      },
+      preferredUnitTypes: [
+        {
+          id: unitTypeA.id,
+        },
+      ],
+      householdMember: [
+        {
+          orderId: 0,
+          firstName: 'example first name',
+          middleName: 'example middle name',
+          lastName: 'example last name',
+          birthMonth: '12',
+          birthDay: '17',
+          birthYear: '1993',
+          sameAddress: YesNoEnum.yes,
+          relationship: 'example relationship',
+          workInRegion: YesNoEnum.yes,
+          householdMemberWorkAddress: exampleAddress,
+          householdMemberAddress: exampleAddress,
+        },
+      ],
+      appUrl: 'http://www.example.com',
+      additionalPhone: true,
+      additionalPhoneNumber: '111-111-1111',
+      additionalPhoneNumberType: 'example type',
+      householdSize: 2,
+      housingStatus: 'example status',
+      sendMailToMailingAddress: true,
+      householdExpectingChanges: false,
+      householdStudent: false,
+      incomeVouchers: false,
+      income: '36000',
+      incomePeriod: IncomePeriodEnum.perYear,
+      language: LanguagesEnum.en,
+      acceptedTerms: true,
+      submissionDate: submissionDate,
+      reviewStatus: ApplicationReviewStatusEnum.valid,
+      programs: [
+        {
+          key: 'example key',
+          claimed: true,
+          options: [
+            {
+              key: 'example key',
+              checked: true,
+              extraData: [
+                {
+                  type: InputType.boolean,
+                  key: 'example key',
+                  value: true,
+                },
+              ],
+            },
+          ],
+        },
+      ],
+    };
+
+    const res = await request(app.getHttpServer())
+      .post(`/applications/verify`)
+      .send(dto)
+      .expect(201);
+
+    expect(res.body.success).toEqual(true);
+  });
+
+  it('should throw an error when trying to verify an incomplete application', async () => {
+    const dto: ApplicationCreate = {} as unknown as ApplicationCreate;
+
+    await request(app.getHttpServer())
+      .post(`/applications/verify`)
+      .send(dto)
+      .expect(400);
```

Comment:
Can you add an expect statement that the response body has the right message?

Reply:
updated!